en/deploy/README.md

```diff
@@ -8,7 +8,16 @@ As you learned, a website has to be located on a server. There are a lot of serv
 
 The other external service we'll be using is [GitHub](http://www.github.com), which is a code hosting service. There are others out there, but almost all programmers have a GitHub account these days, and now so will you!
 
-We'll use GitHub as a stepping stone to transport our code to and from PythonAnywhere.
+We'll use GitHub as a stepping stone to transport our code to and from PythonAnywhere.  Once the initial setup is complete, your workflow will look like:
+
+1. Modify your program on your own computer
+2. Test your program (`python manage.py runserver`)
+3. Save your work in your local change tracking system, called "git" (see below)
```

Comment:
I'd use phrases they are more familiar with, so maybe, "Commit your changes to Git using ```git add -A .``` and ```git commit -m "[Commit message]"```